overlord/ifacestate/handlers.go

```diff
@@ -102,7 +102,7 @@ func (m *InterfaceManager) setupProfilesForSnap(task *state.Task, _ *tomb.Tomb,
 	// - restore connections based on what is kept in the state
 	//   - if a connection cannot be restored then remove it from the state
 	// - setup the security of all the affected snaps
-	affectedSnaps, err := m.repo.DisconnectSnap(snapName)
+	disconnectAffected, err := m.repo.DisconnectSnap(snapName)
```

Comment:
Before I could tell this was a list of snaps that were touched. Now it can be pretty much anything.

How about `disconnectedSnaps` and `connectedSnaps` below?